MK3 sync Prusa-Error-Codes

3d-gussner opened the pull request:
Added script `check-pce.sh` to check if MK3 MMU messages are same as [PCE MMU](https://github.com/prusa3d/Prusa-Error-Codes/blob/master/04_MMU/error-codes.yaml) 

Please also review https://github.com/prusa3d/Prusa-Error-Codes/pull/93

@DRracer The error code `04900` is in PCE but not in MK3 branch

3d-gussner committed:
Sync PEC `FINDA_DIDNT_GO_OFF` to `FINDA_FILAMENT_STUCK`

3d-gussner committed:
Sync PEC `FSENSOR_DIDNT_GO_OFF` to `FSENSOR_FILAMENT_STUCK`

3d-gussner committed:
Sync PEC `PULLEY_STALLED` to `PULLEY_CANNOT_MOVE`

3d-gussner committed:
Sync PEC `SELECTOR_CANNOT_MOVE` update text

3d-gussner committed:
Sync PEC `IDLER_CANNOT_MOVE` update text

3d-gussner committed:
Sync PEC `PULLEY_WARNING_TMC_TOO_HOT` to `WARNING_TMC_PULLEY_TOO_HOT`

3d-gussner committed:
Snyc PEC `RestartMMU` to `ResetMMU`
Also related text

3d-gussner committed:
Sync PEC `SELECTOR_WARNING_TMC_TOO_HOT` to `WARNING_TMC_SELECTOR_TOO_HOT`

3d-gussner committed:
Sync PEC `IDLER_WARNING_TMC_TOO_HOT` to `WARNING_TMC_IDLER_TOO_HOT`

3d-gussner committed:
Sync PEC `PULLEY_TMC_OVERHEAT_ERROR` to `TMC_PULLEY_OVERHEAT_ERROR`

3d-gussner committed:
Sync PEC `SELECTOR_TMC_OVERHEAT_ERROR` to `TMC_SELECTOR_OVERHEAT_ERROR`

3d-gussner committed:
Sync PEC `IDLER_TMC_OVERHEAT_ERROR` to `TMC_IDLER_OVERHEAT_ERROR`

3d-gussner committed:
Sync PEC `PULLEY_TMC_DRIVER_ERROR` to `TMC_PULLEY_DRIVER_ERROR`
Also releated text

3d-gussner committed:
Sync PEC `SELECTOR_TMC_DRIVER_ERROR` to `TMC_SELECTOR_DRIVER_ERROR`
Also related text

3d-gussner committed:
Sync PEC `IDLER_TMC_DRIVER_ERROR` to `TMC_IDLER_DRIVER_ERROR`
Also related text

3d-gussner committed:
Sync PEC `PULLEY_TMC_DRIVER_RESET` to `TMC_PULLEY_DRIVER_RESET`

3d-gussner committed:
Sync PEC `SELECTOR_TMC_DRIVER_RESET` to `TMC_SELECTOR_DRIVER_RESET`

3d-gussner committed:
Sync PEC `IDLER_TMC_DRIVER_RESET` to `TMC_IDLER_DRIVER_RESET`

3d-gussner committed:
Sync PEC `PULLEY_TMC_UNDERVOLTAGE_ERROR` to `TMC_PULLEY_UNDERVOLTAGE_ERROR`

3d-gussner committed:
Sync PEC `SELECTOR_TMC_UNDERVOLTAGE_ERROR` to `TMC_SELECTOR_UNDERVOLTAGE_ERROR`

3d-gussner committed:
Sync PEC `IDLER_TMC_UNDERVOLTAGE_ERROR` to `TMC_IDLER_UNDERVOLTAGE_ERROR`

3d-gussner committed:
Sync PEC `PULLEY_TMC_DRIVER_SHORTED` to `TMC_PULLEY_DRIVER_SHORTED`
Also related text

3d-gussner committed:
Sync PEC `SELECTOR_TMC_DRIVER_SHORTED` to `TMC_SELECTOR_DRIVER_SHORTED`
Also related text

3d-gussner committed:
Sync PEC `IDLER_TMC_DRIVER_SHORTED` to `TMC_IDLER_DRIVER_SHORTED`
Also related text

3d-gussner committed:
Sync PEC `PULLEY_SELFTEST_FAILED` to `MMU_PULLEY_SELFTEST_FAILED`
Also related text

3d-gussner committed:
Sync PEC `SELECTOR_SELFTEST_FAILED` to `MMU_SELECTOR_SELFTEST_FAILED`
Also releated text

3d-gussner committed:
Sync PEC `IDLER_SELFTEST_FAILED` to `MMU_IDLER_SELFTEST_FAILED`
Also related text

3d-gussner committed:
Sync PEC fix MMU_MMU
Update MMU MCU UNDERPOWER

3d-gussner committed:
Sync PEC `FIL_ALREADY_LOADED` to `FILAMENT_ALREADY_LOADED`
Also related text

3d-gussner committed:
Add Prusa Error Codes check script

3d-gussner committed:
Remove `:` at FINDA and FSENSOR as it is in other tiles

DRracer commented on lang/po/Firmware_cs.po:
```diff
@@ -2470,7 +2470,7 @@ msgstr ""
 #. MSG_TITLE_MCU_UNDERVOLTAGE_VCC c=20
 #: ../../Firmware/mmu2/errors_list.h:166 ../../Firmware/mmu2/errors_list.h:211
-msgid "MCU UNDERVOLTAGE VCC"
+msgid "MMU MCU UNDERPOWER"
```
we may need to update the translations in this case

3d-gussner replied:
Working on this.

3d-gussner replied:
BTW Undervoltage != underpower

Voltage != power

DRracer reviewed (approved):
nice :+1: , well structured

3d-gussner committed:
Sync
FSENSOR_DIDNT_TRIGGER
MSG_DESC_SELECTOR_CANNOT_MOVE
MSG_DESC_IDLER_CANNOT_MOVE
MSG_DESC_FW_UPDATE_NEEDED
FILAMENT_ALREADY_LOADED

3d-gussner committed:
Add MMU UNKOWN_ERROR PEC

3d-gussner commented:
@DRracer Please review newest changes.

DRracer reviewed (approved):
well done :+1:
